src/web/components/SharingPermission/ParticipantItem.tsx

```diff
@@ -6,27 +6,51 @@ import { TriStateCheckbox } from '../Core/TriStateCheckbox';
 
 import './ParticipantItem.scss';
 
-type ParticipantItemProps = {
+function getParticipantTypes(participantTypes?: z.infer<typeof ParticipantTypeSchema>[]) {
+  if (!participantTypes) return null;
+  return participantTypes.map((pt) => (
+    <div className='participant-type-label' key={pt.typeName}>
+      {pt.typeName}
+    </div>
+  ));
+}
+
+type ParticipantItemSimpleProps = {
   participant: AvailableParticipantDTO;
+};
+
+export function ParticipantItemSimple({ participant }: ParticipantItemSimpleProps) {
+  const logo = '/default-logo.svg';
+
+  return (
+    <>
```

Comment:
ideally we'd stick a classname here but we can't since we can't put a <div> here. this means that we lose some structure in the scss